config.py

```diff
@@ -1479,9 +1505,72 @@ def get_sessions():
         return sessions_
 
 
-def get_runs() -> list:
+def get_runs_all_subjects() -> dict:
+    '''Returns a dict of runs present in the bids_path
+    for each subject asked in the configuration file
+    (and not for each subject present in the bids_path).'''
```

Comment:
Can you use double-quotes " around the docstring please and make it such that you have a short one-sentence summary in the first line? The docstring must look sth like this:

```
"""Do this and that.

More detailed explanation here!
"""
```

You also don't need to mention the return types there – ideally, add a `Returns` section for this purpose!